custom_components/tuya_local/devices/EM3390TF_weather_station.yaml

```diff
@@ -0,0 +1,50 @@
+name: Weather Station
+products:
+  - id: cvurn5ynloebrpwy
+    name: EM3390TF

```

Comment:
It would be better to include the brand name as well as the model number here.

Reply:
I also thought about adding the brand at the begginning, but his is one of those generic chinese products that are sold under different brands. I bought it as "VIFLYKOO", but if you search in google you can find listings as "UZOLI " or "JELY"  brand for example, so IDK if it makes sense to add the brand here, since it might be confusing for people with the same model under a different brand.

Reply:
Yes I found the Uzoli and Jely branded ones in my search (but not this one).  I am not sure whether the product id will be the same or different for the other brands. But since this is only the name under products, and the top level name is the one that will be most visible, I don't think it is too misleading, and if we later find out the other brands use the same product id, then all 3 brands can be listed.